cards/tests/dominion/test_adventurer.py

```diff
@@ -0,0 +1,38 @@
+from contextlib import contextmanager
+
+from cards.kingdom_cards.dominion import Adventurer
+from games.factories import GameFactory
+from players.factories import PlayerFactory
+from testing import BaseTestCase
+from turns.factories import TurnFactory
+
+
+class AdventurerCardTestCase(BaseTestCase):
+    def setUp(self):
+        super().setUp()
+        self.player = PlayerFactory()
+        self.game = GameFactory(players=[self.player] + PlayerFactory.create_batch(11))
```

Comment:
Any reason in particular you are dumping 12 players in the game? Lol

Reply:
Do I have a reason? No. Did you have a reason? Probably not. Copied from another test. lol

Reply:
Did you copy from Witch? For the witch it makes sense to have more players than there are curses. And since we are always using 10 curses right now...